benchmarks/cpp/nvfuser/matmul.cpp

```diff
@@ -0,0 +1,356 @@
+#include <torch/csrc/jit/codegen/cuda/arith.h>
```

Comment:
Note to myself: I have split this file out and merged separately in https://github.com/csarofeen/pytorch/pull/2007